lib/mri/webrick/server.rb

```diff
@@ -258,18 +258,26 @@ def run(sock)
     # the client socket.
 
     def accept_client(svr)
-      sock = nil
-      begin
-        sock = svr.accept
-        sock.sync = true
-        Utils::set_non_blocking(sock)
-      rescue Errno::ECONNRESET, Errno::ECONNABORTED,
-             Errno::EPROTO, Errno::EINVAL
-      rescue StandardError => ex
-        msg = "#{ex.class}: #{ex.message}\n\t#{ex.backtrace[0]}"
-        @logger.error msg
+      case sock = svr.to_io.accept_nonblock(exception: false)
```

Comment:
Do we support `accept_nonblock`?

Reply:
We do!

Unfortunately it isn't declared as taking any arguments, so I'll need to fix that.